chia/rpc/wallet_rpc_api.py

```diff
@@ -52,6 +52,7 @@ def get_routes(self) -> Dict[str, Callable]:
         return {
             # Key management
             "/log_in": self.log_in,
+            "/get_fingerprint": self.get_fingerprint,
```

Comment:
Do we feel that the name `get_fingerprint` provides enough context to indicate that it returns the logged-in key's fingerprint? The name strikes me as being a bit general. If I were looking at the available APIs at a glance, I don't think my first thought would be that this indicates the currently logged-in wallet key. I could be an outlier though.

Reply:
I changed the name to `get_logged_in_fingerprint` :+1: